packages/cypress/src/add-commands/index.ts

```diff
@@ -1,3 +1,5 @@
+import { Matcher as TMatcher } from '@testing-library/dom';
+import 'cypress-real-events';
```

Comment:
I suppose all the type checking errors come from adding this here. I noticed though that type checking works fine when we add this in `cypress/support/commands.ts` as stated [in docs](https://github.com/dmtrKovalenko/cypress-real-events#installation). Obviously we need to import in the package but I'm not sure how.

Reply:
That's more for the consumers, here we are providing commands

Reply:
I'm sorry but I don't understand what we need to do here.
We are using the `cypress-real-events` within this package (using the `realHover` command) so I think we need it.

> That's more for the consumers, here we are providing commands
Do you mean this should be a peer dependency?